solar_system.rb

```diff
@@ -0,0 +1,76 @@
+
+
+class SolarSystem
+  attr_reader :solar_system_name, :planets
+
+  def initialize(solar_system_name)
+    @solar_system_name = solar_system_name
+    @planets = []
+  end
+
+  def add_planet(planet)
+    @planets << planet
+  end
+
+  def get_info_new_planet
+    print "\nWhat is your new Planet's name?"
+    planet_name = gets.chomp.to_s
+    print "\nWhat is #{planet_name}'s color?"
```

Comment:
You have a lot of code that interacts with the user here, which makes me think this method might fit better in `main.rb`.